README-SI.md

```diff
@@ -160,7 +158,6 @@ GitHub හි දායක වීමට සුදුසු issues වෙත ස
 ## විවෘත මූලාශ්‍ර දායක මුලපිරීම්
 
 - [Up For Grabs](https://up-for-grabs.net/) - නවකයින්ට ගැලපෙන issues ඇතුළත් ව්‍යාපෘති ඇත
-- [First Timers Only](https://www.firsttimersonly.com/) - "first-timers-only" ලෙස ලේබලගත කළ දෝෂවල ලැයිස්තුවක්.

```

Comment:
This can be kept and added back in.

Reply:
why are we using this link for two time

Reply:
add66b602f87077fb9a28bfc9c24e09203be26b5 done